libiocage/lib/helpers.py

```diff
@@ -103,6 +103,47 @@ def validate_name(name):
     return bool(validate.fullmatch(name))
 
 
+# data -> bool | default
+def parse_bool(data, default=False):
+    """
+    try to parse booleans from strings
+
+    On success, it returns the parsed boolean
+    on failure it returns the `default`.
+    By default, `default` is `False`.
+
+    >>> parse_bool("YES")
+    True
+    >>> parse_bool("false")
+    False
+    >>> parse_bool("-")
+    False
```

Comment:
`-` should resolve to `None`. We also require this behavior in try_parse_bool later on.